internal/pkg/pluginengine/change.go

```diff
@@ -96,6 +101,14 @@ func execute(smgr statemanager.Manager, changes []*Change) map[string]error {
 		var err error
 		var returnValue map[string]interface{}
 
+		log.Infof("Tool's raw changes are: %s.", c.Tool.Options)
```

Comment:
The `Options` here is a map, so what will happen with printing it as a string(%s)?

Reply:
will print the options without true values

![image](https://user-images.githubusercontent.com/94163234/158929334-005d87a4-7707-4140-b5f7-08e7f828ef9f.png)